src/api/login/login.controller.ts

```
@@ -44,11 +46,33 @@ export class LoginController {
   async kakaoLoginCallback(@Query('code') code: string, @Res() res: Response) {
     const result = await this.loginService.processKakaoLogin(code, res);
 
-    const redirectUrl = new URL(process.env.FRONTEND_LOGIN_CALLBACK_URL);
+    const redirectUrl = new URL(process.env.FRONTEND_LOGIN_CALLBACK_URL as string);
```

ESLint

[error] Replace `process.env.FRONTEND_LOGIN_CALLBACK_URL·as·string` with `⏎······process.env.FRONTEND_LOGIN_CALLBACK_URL·as·string,⏎····`

(prettier/prettier)